Src/BootCamp.Chapter/Player.cs

```diff
@@ -12,50 +12,58 @@ namespace BootCamp.Chapter
     /// </summary>
     public class Player
     {
-        /// <summary>
-        /// Everyone can carry this much weight at least.
-        /// </summary>
-        private const int baseCarryWeight = 30;
+        private void BonusTask()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'BonusTask'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5148522)